### PR TITLE
Feat resumable upload swift

### PR DESCRIPTION
**Pull request by abnegate**

### CODE REVIEW

File: templates/swift/Sources/Client.swift.twig [hunk 1]
```diff
@@ -228,7 +229,7 @@ open class Client {
         headers: [String: String] = [:],
         params: [String: Any?] = [:],
         sink: ((ByteBuffer) -> Void)? = nil,
-        convert: (([String: Any]) -> T)? = nil,
+        converter: (([String: Any]) -> T)? = nil,
```
Comment (lohanidamodar) [inline templates/swift/Sources/Client.swift.twig:232]: Why this change?

**Review by lohanidamodar — COMMENTED** [review]
Small query, rest looks good!

File: templates/swift/Sources/Client.swift.twig (hunk 1 above)
Comment (abnegate) [reply templates/swift/Sources/Client.swift.twig:232]: Just a clearer name for the parameter, it is only used internally

**Review by christyjacob4 — APPROVED** [review]
